packages/portfolio-api/docs/ymax-machine.yaml

```diff
@@ -1,130 +1,933 @@
+          observedFrom:
+            - Agoric tx history showing RecvPacket
+          protocol: IBC
+        on:
+          ack.received:
+            target: completed
+            description: Transfer acknowledged successfully
+          ack.error:
+            target: failed
+            description: Agoric returned error acknowledgement
+      completed:
+        description: IBC transfer complete; funds on Agoric LCA
+        type: final
+        meta:
+          row: Agoric Chain
+          observedFrom:
+            - Agoric LCA balance increased
+            - Noble ICA balance decreased
+      timed_out:
+        description: IBC packet timed out; funds refunded on Noble
+        type: final
+        meta:
+          row: Noble Chain
+          severity: error
+          observedFrom:
+            - MsgTimeout on Noble
+            - Noble ICA balance restored
+      failed:
+        description: IBC transfer failed
+        type: final
+        meta:
+          row: Noble Chain
+          severity: error
+          observedFrom:
+            - FlowStatus state=fail with error message
+
+  # =============================================================================
+  # STEP MACHINES - CCTP (Circle Cross-Chain Transfer Protocol)
+  # =============================================================================
+
+  CCTP_noble_EVM:
+    description: CCTP transfer from Noble to EVM chain. Burns USDC on Noble, Circle attestation, mint on EVM.
+    category: step
+    initial: initiated
+    states:
+      initiated:
+        description: noble_ica.depositForBurn(destinationAddress, denomAmount) called
+        meta:
+          row: Noble Chain
+          observedFrom:
+            - FlowStatus state=run with how=CCTP
+          txType: CCTP_TO_EVM
+          protocol: Circle CCTP
+        on:
+          burn.submitted:
+            target: burn_pending
+            description: depositForBurn ICA tx submitted
+          burn.failed:
+            target: failed
+            description: depositForBurn submission failed
+      burn_pending:
+        description: Waiting for Noble to confirm burn transaction
+        meta:
+          row: Noble Chain
+          observedFrom:
+            - Noble tx history showing depositForBurn
+            - resolver pendingTxs with type=CCTP_TO_EVM, status=pending
+          protocol: Circle CCTP
+        on:
+          burn.confirmed:
+            target: attestation_pending
+            description: Burn tx confirmed on Noble; waiting for Circle attestation
+          burn.error:
+            target: failed
+            description: Burn tx failed on Noble
+      attestation_pending:
+        description: Burn confirmed; waiting for Circle attestation service to sign
+        meta:
+          row: Circle Attestation
+          observedFrom:
+            - Circle Attestation API (iris-api.circle.com/v1/attestations)
+          observabilityTodo: "TODO: Poll Circle attestation API and publish status"
```

Comment:
what's the difference between `observedFrom ... Noble tx history showing depositForBurn` and `observabilityTodo: "TODO: Poll Circle attestation API and publish status"` ? we don't do either, currently, IIUC.